Ex1/week5_3.py

```diff
@@ -0,0 +1,33 @@
+# לחששנית
+import re
+
+
+def find_all_hidden_messages_in_a_binary_file_via_pattern(path, pattern):
+    """
+    The function gets a path to a binary file, and a binary pattern and search all "strings" that
+    matches the pattern (those are coded messages in the file) then returns a list of those messages
+    as regular strings.
+    :return: A list of the hidden messages
+    """
+    reg = re.compile(pattern)
+    with open(path, "rb") as encrypted_binary_file:
+        return [''.join(msg) for msg in [hidden_msg.decode('ascii')
```

Comment:
use double quotes "" and not single '' when you create strings, unless you have a " inside